modules/users/operator.nix

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+_: {
+  users.users = {
+    operator = {
```

Comment:
Should it not still be called Ghaf?

Reply:
Maybe `admin` can be renamed as `ghaf`, so it won't impact much on functionality?